phpBB/styles/prosilver/theme/stylesheet.css

```diff
@@ -19,4 +19,5 @@
 @import url("icons.css?v=3.3");
 @import url("colours.css?v=3.3");
 @import url("responsive.css?v=3.3");
+@import url("bidi.css?v=3.3");
```

Comment:
Current 3.2 has bidi.css loading AFTER stylesheet.css so this is in the correct spot. Also this is a simple stylesheet. Moving its contents out to say overall_header.html won't change that for those that are "confused". Plus it would mean that each item would need to be included in each file that currently includes just the one stylesheet.css file.

Reply:
He means not have a bidi.css but change per component. This is what we are doing in the new theme but quite impossible in the scope of prosilver